client/gutenberg/extensions/publicize/editor.scss

```diff
@@ -30,19 +30,19 @@ $dark-gray-500: #555d66;
 	margin-right: 5px;
 
 	&.is-facebook {
-		fill: #3957a2;
+		fill: var( --color-facebook );
 	}
 	&.is-twitter {
-		fill: #55acee;
+		fill: var( --color-twitter );
 	}
 	&.is-linkedin {
-		fill: #0976b4;
+		fill: var( --color-linkedin );
 	}
 	&.is-tumblr {
-		fill: #35465c;
+		fill: var( --color-tumblr );
 	}
 	&.is-google-plus {
-		fill: #df4a33;
+		fill: var( --color-gplus );
```

Comment:
I noticed that these use HEX codes...is there a specific reason why? I'm assuming it can just be set to variables too. 
(cc @floor, @blowery, @drw158)

Reply:
Agreed. Are all the hex values the same?

Reply:
These ones are, but the ones [here](https://github.com/Automattic/wp-calypso/blob/2d50a28f61230d2d2a2ab71088241ab7a056fc05/client/blocks/reader-share/style.scss#L75-78) are a bit different. The HEX code seem are  identical visually to the actual social media icons though, so I'm assuming they also should use a variable.